react/introduction/setting_up_a_react_environment.md

```diff
@@ -0,0 +1,145 @@
+### Introduction
+
+You now know _what_ React is, you might even have an idea of _where_ to use it, it's time to start figuring out _how_. 
+
+This lesson is going to cover starting a new React project on your own machine, as well as some useful tools to help you along the way. We'll also explain some of the problems that may arise (and how to avoid them).
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+* How React projects can be created locally.
+* Describing the difficulties when starting a React project without the appropriate tools.
+* Listing some of the popular tools and methods for scaffolding a new React project.
+* Explaining what Create React App is.
+* Setting up your workflow to develop efficiently within your new project.
+
+### Many Paths
+
+There are multiple ways to start using React in your projects, from attaching a set of `<script>` tags which serve React from a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network), to robust [toolchains](https://en.wikipedia.org/wiki/Toolchain) that are highly configurable and allow for increased scalability and optimization.
+
+Some examples of these toolchains:
+
+* [Create React App](https://create-react-app.dev/)
+* [Vite's React Config](https://vitejs.dev/)
+* [Gatsby](https://www.gatsbyjs.com/)
+* [NextJS](https://nextjs.org/)
```

Comment:
Gatsy and NextJS are more like frameworks rather than toolchains. But we could accommodate to keep these in.

```suggestion
There are multiple ways to start using React in your projects, from attaching a set of `<script>` tags which serve React from a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network), to robust [toolchains](https://en.wikipedia.org/wiki/Toolchain) and [frameworks](https://en.wikipedia.org/wiki/Framework) that are highly configurable and allow for increased scalability and optimization.

Some examples:

* [Create React App](https://create-react-app.dev/)
* [Vite's React Config](https://vitejs.dev/)
* [Gatsby](https://www.gatsbyjs.com/)
* [NextJS](https://nextjs.org/)
```

Reply:
Like this a lot, will link directly to the Web Framework page for context.